scripts/check_data_injection.php

```diff
@@ -20,6 +20,9 @@ function grab_dump($var){
 	$logBody = "////////////////////\n".date("Y-m-d H:i:s")."\n";
 	$logBody .= $output."";
 	//$strOut = array();
+	putenv("http_proxy=http://gs600-squid.ndc.nasa.gov:443");
+	putenv("https_proxy=http://gs600-squid.ndc.nasa.gov:443");
+	putenv("http_export=http://gs600-squid.ndc.nasa.gov:443");
```

Comment:
So all of these are specific to running at GFSC. Maybe each of these variables should be in a config file somewhere?

Reply:
They're in a config file now, check the latest code